suites/quincy/cephadm/tier-4-service-operations.yaml

```diff
@@ -140,3 +140,9 @@ tests:
           custom_configs:
             - mount_path: /usr/share/mon/example.cert
               content: test
+
+  - test:
```

Comment:
Hi Pranav,

Can you please add same test for squid suite as well.

Rest look good

Reply:
@MohitBis Yes added for squid as well. Thanks